src/styles/divider.css

```diff
@@ -0,0 +1,10 @@
+/* divider */
+
+.ui.divider:not(.vertical):not(.horizontal) {
```

Comment:
This seems like a weird selector. If it's not vertical nor horizontal, what styles are applied to? Isn't it horizontal by default?

Reply:
It looks like there's just `.ui.divider` selector. So maybe we can move some styles there? And then just put `bordrer` in this selector with higher specifity to override border only?

<img width="250" alt="Screenshot 2023-06-07 at 11 28 11 AM" src="https://github.com/abby-moffitt/GA-Semantic/assets/64768811/9a2fd566-a317-42c4-8cf4-8eb82beb3496">

Reply:
Might be also worth checking if styles are applied to all variants, eg. _horizontal_, _vertical_, _section_, etc.

Reply:
So this `.ui.divider:not(.vertical):not(.horizontal)` was the selector used by default by Semantic UI to define the borders of the div, and it was overriding the `.ui.divider` selector I was using.

Reply:
`horizontal` and `vertical` are variants that allow the divider to segment content horizontally and vertically.

Reply:
like this: 
![image](https://github.com/abby-moffitt/GA-Semantic/assets/101662434/4f4a03ab-1027-444c-b2c0-83e6508af56c)